src/lib/features/access/createAccessService.ts

```diff
@@ -45,7 +45,7 @@ export const createFakeAccessService = (
     const accountStore = new FakeAccountStore();
     const roleStore = new FakeRoleStore();
     const environmentStore = new FakeEnvironmentStore();
-    const accessStore = new FakeAccessStore();
+    const accessStore = new FakeAccessStore(roleStore);
```

Comment:
@kwasniew I'm reusing your `createAccessService` method which simplifies the test setup, but note I'm explicitly adding this dependency because access-store directly queries roles table: https://github.com/Unleash/unleash/blob/0457ae42ee78e9481b295bae3161d049051ededa/src/lib/db/access-store.ts#L23

To simulate this DB coupling I'm adding this dependency.

I couldn't find a better way which is why I'm tagging you in case you have some ideas